src/rgw/rgw_perf_counters.h

```diff
@@ -99,3 +108,20 @@ void tinc(const CountersContainer &counters, int idx, utime_t);
 void tinc(const CountersContainer &counters, int idx, ceph::timespan amt);
 
 } // namespace rgw::op_counters
+
+namespace rgw::persistent_topic_counters {
+
+class CountersContainer {
+  std::shared_ptr<PerfCounters> topic_counters;
```

Comment:
this can be a unique_ptr rather than a shared_ptr

Reply:
Ack!